Makefile.work

```diff
@@ -153,6 +162,31 @@ ifneq ($(SONIC_DPKG_CACHE_SOURCE),)
 	DOCKER_RUN += -v "$(SONIC_DPKG_CACHE_SOURCE):/dpkg_cache:rw"
 endif
 
+
+# User name and tag for "docker-*" images created by native dockerd mode.
+ifeq ($(strip $(SONIC_CONFIG_USE_NATIVE_DOCKERD_FOR_BUILD)),y)
+DOCKER_USERNAME = -$(shell echo $(USER) | tr '[:upper:]' '[:lower:]')
```

Comment:
please use USER_LC, it is ok the use the new name DOCKER_USERNAME, but please consolidate.

Reply:
The naming convention of DOCKER_USERNAME is inlined with DOCKER_USERTAG and DOCKER_LOCKDIR.  If we want to change it to USER_LC, then please suggest a name for other variables also.

Reply:
DOCKER_USERNAME is fine. can you rename USER_LC

Reply:
I have reused the USER_LC.